lisa/tools/reboot.py

```diff
@@ -53,11 +52,21 @@ def reboot(self) -> None:
             sleep(wait_seconds)
             current_delta = date.current().replace(tzinfo=None) - current_boot_time
 
+        command_result = self.node.execute(
+            "command -v sudo", shell=True, no_info_log=True
+        )
+        if command_result.exit_code != 0:
+            raise LisaException("command sudo doesn't exist")
```

Comment:
If the exception is wrotten in sudo.py, it can be like this way. But it's wrotten in reboot, so it should specify what's business logic here. `the system doesn't support sudo, cannot perform reboot.`